app/customer.py

```diff
@@ -0,0 +1,49 @@
+import json
+import math
+from dataclasses import dataclass
+from app.car import fuel_price
+from app.shop import Shop
+
+
+@dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: list
+    money: float
+    car: dict
+
+    def distance(self, shop: Shop) -> float:
+        pow_x_coordinate = pow(self.location[0] - shop.location[0], 2)
+        pow_y_coordinate = pow(self.location[1] - shop.location[1], 2)
+        return math.sqrt(pow_x_coordinate + pow_y_coordinate)
+
+    def price_trip(self, shop: Shop) -> float:
+        fuel_consumption = self.car["fuel_consumption"]
+        distance = self.distance(shop)
+        price_road = 2 * (fuel_consumption / 100) * distance * fuel_price
+
+        price_product = 0
+        for product_name in self.product_cart.keys():
+            count_product = self.product_cart[product_name]
+            price = shop.products[product_name]
+            price_product += count_product * price
+
+        return round(price_road + price_product, 2)
+
+
+with open("app/config.json", "r") as file:
+    customers = json.load(file)["customers"]
+
+customers_list = []
+
+for customer in customers:
+    customers_list.append(
+        Customer(
+            name=customer["name"],
+            product_cart=customer["product_cart"],
+            location=customer["location"],
+            money=customer["money"],
+            car=customer["car"]
+        )
+    )

```

Comment:
Do it as a list comprehension
Also, move it to main.py not good to do it not in function
The same with shops